mlir/tools/mlir-pet/Test/ternary.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: mlir-pet %S/Inputs/ternary.c | FileCheck %s
```

Comment:
The test case is correct but it is difficult to understand. Maybe we should just be explicit on the SSA values?